src/main/kotlin/com/autonomousapps/tasks/XmlSourceExploderTask.kt

```diff
@@ -194,9 +194,9 @@ private class AndroidResParser(
 
 private class AndroidResBuilder(private val relativePath: String) {
 
-  val styleParentRefs = mutableSetOf<AndroidResSource.StyleParentRef>()
-  val attrRefs = mutableSetOf<AndroidResSource.AttrRef>()
-  val usedClasses = mutableSetOf<String>()
+  val styleParentRefs = sortedSetOf<AndroidResSource.StyleParentRef>()
+  val attrRefs = sortedSetOf<AndroidResSource.AttrRef>()
+  val usedClasses = sortedSetOf<String>()
```

Comment:
Noticed that these sets weren't properly sorted, which could result in inconsistent outputs between task invocations.